collection-scripts/gather_serving

```diff
@@ -2,6 +2,12 @@
 # shellcheck disable=SC1091
 source common.sh
 resources=("inferenceservices" "inferencegraphs" "trainedmodels" "servingruntimes" "clusterstoragecontainers" "predictors")
+# Dependent resources ossm
+resources+=("smcp" "smm" "smmr" "gateways" "authorizationpolicies" "envoyfilters")
+# Dependent resources knative
+resources+=("knativeservings")
```

Comment:
The `knativeservice` is missing

Reply:
Yes, for must-gather, we probably should be collecting all Knative related resources:
* services, configurations, routes, revisions
* Everything under serving.knative.dev API group.

That will make easier for the Serverless team to assist us, if needed.

Reply:
updated

i got confused by the resources from openshift core and knative.
speicified with api resource name